NwPluginAPI/Core/Extensions/ItemExtensions.cs

```diff
@@ -0,0 +1,220 @@
+using InventorySystem;
+using InventorySystem.Items;
+using InventorySystem.Items.Firearms;
+using InventorySystem.Items.Firearms.Attachments;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PluginAPI.Core.Extensions
+{
+	public static class ItemExtensions
+	{
+		/// <summary>
+		/// Gets if the <see cref="ItemType"/> is ammo for a firearm.
+		/// </summary>
+		/// <param name="type">The <see cref="ItemType"/> to check</param>
+		/// <returns>Returns whether or not the <see cref="ItemType"/> is ammo.</returns>
+		public static bool IsAmmo(this ItemType type)
```

Comment:
Have you seen a use for these kinds of methods?
Generally speaking one could use the ItemBase with an 'is Firearm'
Although it could potentially be used for some kind of config/command validation, please comment your opinion.

Reply:
Normally I use it when I have to give the inventory by code and divide the ammunition in another list because they have to be sent with a little delay after the objects, besides if I give ammunition and then give a bodyArmor it will throw the extra ammunition I gave it.

Reply:
I understand, that's fine then.